src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -3986,7 +3851,6 @@ def list_artifacts(
                 query=query,
                 table=ArtifactSchema,
                 filter_model=artifact_filter_model,
-                custom_schema_to_model_conversion=self._artifact_schema_to_model,
```

Comment:
Now that we no longer need these for runs / steps / artifacts, I think we could get rid of the `custom_schema_to_model_conversion` arg in `filter_and_paginate()` entirely.

Reply:
I would rather do this in a separate PR.